packages/wallet-sdk/package.json

```diff
@@ -77,5 +84,6 @@
     "tsc-alias": "^1.8.8",
     "tslib": "^2.6.0",
     "typescript": "^5.1.6"
-  }
-}
+  },
+  "packageManager": "yarn@1.22.19+sha1.4ba7fc5c6e704fce2066ecbfb0b0d8976fe62447"
```

Comment:
btw, do you mind install a global yarn using npm, and run things again so we can remove this line? this packageManager i think is added by the builtin yarn from npm